### PR TITLE
C51-327: Add Open in Popup icon

### PR DESCRIPTION
**Pull request by deb1990**
## Overview
As part of this PR a button has been added in the Activity Details panel, to open the activity in a popup.

## How it looks
after

2018-11-06 at 4 36 pm

## Technical Details
`civicase__activity-panel__maximise` class is added for the button, and using css it has been rotated to get the desired look.

### CODE REVIEW

**Comment by jamienovick** [comment]
Is there a danger or users clicking this or close by mistake with it so close to close? What about the other side?

File: scss/components/_activity-panel.scss [hunk 1]
```diff
@@ -42,12 +42,17 @@
...
-.civicase__activity-panel__close {
+.civicase__activity-panel__close,
+.civicase__activity-panel__maximise {
   color: $gray-darker;
   font-size: $font-size-h2;
   padding: 0;
 }
 
+.civicase__activity-panel__maximise {
+  transform: rotate(45deg);
```
Comment (swastikpareek) [inline scss/components/_activity-panel.scss]: Nice trick 😉

**Comment by swastikpareek** [comment]
> Is there a danger or users clicking this or close by mistake with it so close to close? What about the other side? Would need to test and see?

Good point. But we have a tooltip which says `View in Popup` which will eliminate the accidental click on close button. 
What are your thoughts @deb1990 ?
cc @jamienovick

**Comment by deb1990** [comment]
@jamienovick @JonScreat 
I have added a new ticket [C51-332](https://compucorp.atlassian.net/browse/C51-332) to identify and add the hover state for all icons.
Also I have added a bit more spacing between the icons to reduce the chance of accidental click. Please let me know if this looks good.
2018-11-06 at 4 36 pm

**Comment by swastikpareek** [comment]
Looks good to me. Approved 🥂

**Pull request merged**